frontend/src/config/keybindings/index.ts

```diff
@@ -61,6 +61,23 @@ export function getConfiguredShortcutsForAction(scope: string, action: string):
   return keyBindingsConfig[scope][action];
 }
 
+/**
+ * Retrieves the tooltip string for a specific action within a specified scope, including the associated keyboard shortcut.
+ * @param scope - The scope in which to look for the associated action's shortcut.
+ * @param action - The action for which to find the associated keyboard shortcut.
+ * @param descriptionText - Optional description text to include in the tooltip.
+ * @returns The tooltip string including the keyboard shortcut if found, or just the description text if not found.
+ *         If no description text is provided, the tooltip will only include the keyboard shortcut.
+ *
+ * */
+export function getTooltipStringWithShortcut(
```

Comment:
I would prefer a consistent nomenclature. The module is called `keybindings` and in the issue we talk about keybindings as well. Are "shortcuts" not the same as "keybindings".

Additionally, I think that keybinding is not the right word here.
We pass in an action and we get a text description of the keybinding.

So imo the function should be called `getDescriptionForKeybinding` or something.
Also imo the name of a function should not talk about the type it returns.
The functions return type has this purpose.